src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -414,6 +415,7 @@ protected Size MeasureChild(View view, Size slotSize)
 
 			return ret;
 		}
+		//public static bool _debugTemp1 = true;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5147344)